client/app/scripts/keyboard-controller.coffee

```diff
@@ -2,10 +2,18 @@ ROT = require('./vendor/rot.js/rot')
 Bacon = require('baconjs')
 
 class KeyboardController
+  BINDING_SELECTOR = 'html'
+  onInput: ->
+    keydowns = $(BINDING_SELECTOR)
```

Comment:
Ah yeah, there used to be one for keypress too, but that didn't fire since no characters were printed.